### DISCUSSION TITLE
the UI

### DISCUSSION THREAD

**Discussion by neoapps-dev** [bmaroti9/Overmorrow#48] [General]
the UI is amazing AF in this app :)

hope it will work on iOS (btw, i dont have an iOS phone but just wanna see it everywhere :) )

**Comment by bmaroti9** [comment]
Hi! Thanks for the compliment on the UI!
iOS is not a main priority for a couple of reasons:
- the 100 dollar price tag to make an apple developer account
- the need for a mac to publish it
- the current UI is basically a material-you implementation (google's design language) and it wouldn't look that good on iOS where they have apple's design language.

I might wan't to add it sometime later, but right now i want to focus on getting the android version up to scratch.
